local_dev_files/build_dev_env.sh

```diff
@@ -55,74 +57,45 @@ else
+cd $basedir/aws/vpn/lambda/code
+yarn install && yarn build && yarn postbuild
+
 if [ "$fresh_instance" = true ]; then
   printf "${yellowColor}=> Detected fresh AWS account instance.  Clearing local files...${reset}\n"
   for dir in $basedir/env/cloud/*/; do
     rm -rf "${dir}.terragrunt-cache"
     rm -f "${dir}.terraform.lock.hcl"
   done
-else
-  if [ -z "$MODULE_NAME" ]; then
-    printf "${greenColor}=> Building All Terragrung Modules${reset}\n"
-  else
-    printf "${greenColor}=> Only building ${MODULE_NAME} Terragrunt Module${reset}\n"
-    cd $basedir/env/cloud/$MODULE_NAME
-    terragrunt apply --terragrunt-non-interactive -auto-approve --terragrunt-log-level warn
-    exit 0
-  fi
 fi
 
-printf "${greenColor}...Setting up KMS${reset}\n"
-cd $basedir/env/cloud/kms
-terragrunt apply --terragrunt-non-interactive -auto-approve --terragrunt-log-level warn
-
-printf "${greenColor}...Setting up Network${reset}\n"
-cd $basedir/env/cloud/network
-terragrunt apply --terragrunt-non-interactive -auto-approve --terragrunt-log-level warn
-
-printf "${greenColor}...Setting up Secrets Manager${reset}\n"
-cd $basedir/env/cloud/secrets
-terragrunt apply --terragrunt-non-interactive -auto-approve --terragrunt-log-level warn
-
-printf "${greenColor}...Creating SQS queue${reset}\n"
-cd $basedir/env/cloud/sqs
-terragrunt apply --terragrunt-non-interactive -auto-approve --terragrunt-log-level warn
-
-printf "${greenColor}...Creating SNS queue${reset}\n"
-cd $basedir/env/cloud/sns
-terragrunt apply --terragrunt-non-interactive -auto-approve --terragrunt-log-level warn
-
-printf "${greenColor}...Setting up Redis${reset}\n"
-cd $basedir/env/cloud/redis
-terragrunt apply --terragrunt-non-interactive -auto-approve --terragrunt-log-level warn
-
-printf "${greenColor}...Setting up RDS${reset}\n"
-cd $basedir/env/cloud/rds
-terragrunt apply --terragrunt-non-interactive -auto-approve --terragrunt-log-level warn
-
-printf "${greenColor}...Setting up S3${reset}\n"
-cd $basedir/env/cloud/s3
-terragrunt apply --terragrunt-non-interactive -auto-approve --terragrunt-log-level warn
-
-printf "${greenColor}...Setting up DynamoDB${reset}\n"
-cd $basedir/env/cloud/dynamodb
-terragrunt apply --terragrunt-non-interactive -auto-approve --terragrunt-log-level warn
-
-printf "${greenColor}...Setting up ECR${reset}\n"
-cd $basedir/env/cloud/ecr
-terragrunt apply --terragrunt-non-interactive -auto-approve --terragrunt-log-level warn
-
-printf "${greenColor}...Setting up Lambdas${reset}\n"
-cd $basedir/env/cloud/lambdas
-terragrunt apply --terragrunt-non-interactive -auto-approve --terragrunt-log-level warn
-
-printf "${greenColor}...Setting up VPN Client${reset}\n"
-
-cd $basedir/aws/vpn/lambda/code
-yarn install && yarn build && yarn postbuild
+if [ -z "$MODULE_NAME" ]; then
+  printf "${greenColor}=> Building All Terragrunt Modules${reset}\n"
+  terragrunt run-all apply \
+    --non-interactive --log-level info \
```

Comment:
The log level was changed (from warn to info). Was this intentional?

```suggestion
    --non-interactive --log-level warn \
```

Reply:
Yes, intentional.  With the warn level there was a long period of no STDOUT which could give the impression that the script was completely frozen and not doing anything.  The info level at least provides some idea that it's still chugging away in the background.